helm/values.yaml

```
@@ -47,53 +48,109 @@ config:
       successThreshold: 1
       failureThreshold: 3
 
-    # Enable mutual TLS authentication. Useful when not running in a secure
-    # environment, i.e. when you're running it locally against your own implementation.
-    MUTUAL_TLS_ENABLED: "false"
+    env:
+      # Port number that the inbound (Mojaloop API) HTTP server will listen on
+      INBOUND_LISTEN_PORT: "4000"
```

Comment this out. Something like this.
```suggestion
      # This option is presented for consistency with the .env file. However, for the purposes of this chart, this port is _not_ configurable.
      # INBOUND_LISTEN_PORT: "4000"
```